lib/index.ts

```diff
@@ -29,6 +32,7 @@ function debug(s: string) {
 export interface MavenOptions extends legacyPlugin.BaseInspectOptions {
   scanAllUnmanaged?: boolean;
   reachableVulns?: boolean;
+  callGraphBuilderTimeout?: number; // milliseconds, as accepted by setTimeout
```

Comment:
Maybe better to expose seconds? I would expect the users to have seconds? Will be nice to have it communicated in the name for less confusion

Reply:
I think milliseconds might give better control. Maybe having the unit in the name would be clearer.

Reply:
Gotcha, I think in the CLI it should be exposed to the user as seconds, like other tools that expose timeout as cli args

Reply:
Agreed! I'll make the user facing option expressed in seconds